test-suite/output/Show.out

```diff
@@ -1,10 +1,10 @@
-3 subgoals (ID 31)
+3 subgoals (ID 29)
   
   H : 0 = 0
   ============================
   1 = 1
 
-subgoal 2 (ID 35) is:
+subgoal 2 (ID 33) is:
  1 = S (S m')
-subgoal 3 (ID 22) is:
+subgoal 3 (ID 20) is:
  S (S n') = S m
```

Comment:
For some reason goal numbers have changed (here and in the other changed test).

Reply:
The evar_kind of `_` (`QuestionMark`) is not the same as the one of an implicit argument. This has consequences on what typeclasses does (see `Typeclasses.no_goals_or_obligations`), and, suspectingly in the order in which goals are declared. Something like that.